apps/api/src/test/java/dk/treecreate/api/healthcheckControllerTests/HealthcheckControllerTests.java

```diff
@@ -0,0 +1,54 @@
+package dk.treecreate.api.healthcheckControllerTests;
+
+import dk.treecreate.api.ApiApplication;
+import io.restassured.builder.RequestSpecBuilder;
```

Comment:
I don't think using Rest Assured library is needed here, you could achieve the same with the built-in MockMvc package

We can discuss this during the MMM